### PR TITLE
Convert format strings to literal string interpolation

### PR DESCRIPTION
**Pull request by joshwatson**
This PR converts most (with only a couple exceptions) of the formatted strings in Manticore to follow PEP-0498, literal string interpolation. Since Manticore 0.2.0 requires 3.6+, it makes sense to take advantage of this language feature.

---
This change is [Reviewable](https://reviewable.io/reviews/trailofbits/manticore/1072)

### CODE REVIEW

File: examples/evm/asm_to_smtlib.py [hunk 1]
```diff
@@ -109,22 +109,22 @@ def send_funds(address, recipient, value):
 
 print("CODE:")
 while not issymbolic(evm.pc):
-    print('\t',evm.pc, evm.instruction)
+    print(f'\t {evm.pc} {evm.instruction}')
```
Comment (mossberg) [inline examples/evm/asm_to_smtlib.py]: personally, i prefer the old style of this line :)

**Comment by mossberg** [comment]
thanks @joshwatson :) it might be a little while before we get to reviewing and merging this; we're currently sprinting on a few other features right now in prep for an upcoming release.

**Comment by disconnect3d** [comment]
Suggestion: skip all the conflicting files and merge it?

**Comment by mossberg** [comment]
that sounds good to me 👍

**Comment by disconnect3d** [comment]
Closing in favor of https://github.com/trailofbits/manticore/pull/1194

**Pull request closed without merging**